src/general/Mutations.cs

```diff
@@ -569,23 +572,12 @@ private string MutateWord(string name)
             return MutateWord(name);
         }
 
-        // TODO: C# probably has better ways to handle case conversions
-
         // Convert to lower case
-        for (int i = 1; i < newName.Length - 1; i++)
-        {
-            if (newName[i] >= 65 && newName[i] <= 92)
-            {
-                newName[i] = (char)(newName[i] + 32);
-            }
-        }
+        string lowercase = newName.ToString().ToLower(CultureInfo.CurrentCulture);
```

Comment:
I feel like maybe InvariantCulture should be used to ensure consistency in the species name creation.